RELEASE/scripts/autoscend/auto_combat.ash

```diff
@@ -957,7 +957,11 @@ string auto_combatHandler(int round, monster enemy, string text)
 		return useSkill($skill[Apprivoisez La Tortue], false);
 	}
 
-	if(!in_zelda() && canUse($skill[Gulp Latte]) && (get_property("_latteRefillsUsed").to_int() == 0) && !get_property("_latteDrinkUsed").to_boolean())
```

Comment:
_latteRefillsUsed == 0 check is ensuring we only use gulp latte once per day when it is available 4 times a day. huge waste.

_latteDrinkUsed check should be redundant as we are already using `canUse($skill[Gulp Latte])` which in theory will return false when gulp latte is not available. I think `_latteDrinkUsed` is meant for out of combat checking on whether or not latte is ready to gulp.
I am going to test removing that check tomorrow and just using `canUse`